platform/macos/src/NSColor+MGLAdditions.hpp

```diff
@@ -0,0 +1,17 @@
+#import <Cocoa/Cocoa.h>
+
+#include <mbgl/util/color.hpp>
+
+@interface NSColor (MGLAdditions)
+
+/**
+ Converts the color into an mbgl::Color in calibrated RGB space.
+ */
+- (mbgl::Color)mbgl_color;
+
+/**
+ Instantiates `NSColor` from an `mbgl::Color`
+ */
++ (NSColor *)mbgl_color:(mbgl::Color)color;
```

Comment:
This should be `+mbgl_colorWithColor:` so it’s clear that it initializes a new color object.